hooks/base16-jqp.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# ----------------------------------------------------------------------
+# Setup config variables and env
+# ----------------------------------------------------------------------
+#
+
+BASE16_JQP_CONFIG_FILE="${BASE16_JQP_CONFIG_FILE:-$BASE16_CONFIG_PATH/jqp_theme.yaml}"
```

Comment:
Suggesting this edit since while a lot of other env vars are `BASE16_*`, that's outdated since this repo also has base24 themes, so those need to be changed (in a backward compatible way). Would be good if future env vars are set correctly from the start.

```suggestion
TINTED_JQP_CONFIG_FILE="${BASE16_JQP_CONFIG_FILE:-$BASE16_CONFIG_PATH/jqp_theme.yaml}"
```

Reply:
Done https://github.com/tinted-theming/tinted-shell/pull/57/commits/e2a9e2ea4a7c0cfe7c5e5897a026a5e7f731fb91